quasar.conf.js

```diff
@@ -76,6 +77,8 @@ module.exports = configure(function (ctx) {
         chain.plugin('node-polyfill').use(nodePolyfillWebpackPlugin);
         // Ref: https://quasar.dev/quasar-cli/linter
         chain.plugin('eslint-webpack-plugin').use(ESLintPlugin, [{ extensions: ['js', 'vue'] }]);
+        chain.resolve.symlinks(false);
+        chain.resolve.alias.set('vue', path.resolve('./node_modules/vue'));
```

Comment:
can you tell me what this config is for? Is it because of the compatibility with `@polkadot/vue-identicon`?

Reply:
No, I've followed this branch to install the astar-ui library.
https://github.com/AstarNetwork/astar-apps/pull/292/files#diff-1d7ff4f413e8862d32dfe18796e76517d15cdb33a09e11ce2dc65c07c11df1f6R82

@sirius651 Could you help advise about this question?

Reply:
yes without that config, it encountered error(`Uncaught (in promise) TypeError: Cannot read properties of null (reading 'isCE') :: `) with astar-ui. because it prevent two vue frameworks from being referenced simultaneously.